packages/frontend-shared/src/composables/useMarkdown.ts

```diff
@@ -102,8 +102,15 @@ export const useMarkdown = (target: Ref<HTMLElement>, text: MaybeRef<string>, op
 
     whenever(target, () => {
       useEventListener(target, 'click', (e: MouseEvent) => {
+        const link = (e.target as HTMLElement).closest('a[href]')
```

Comment:
I was not familiar with `.closest` until reviewing this PR, and wow, that is kind of a misnomer. I assumed it searched siblings too. I wish they had called it something like `.closestParent`.

Reply:
I agree with you there. One thing that also isn't immediately clear is that closest() is inclusive of the element that is calling it. So if the target is a link, it'll find itself first. I added tests to cover that case as well.

Reply:
Agree it could be a little wonky but AFAIK it's just the [jQuery  implementation](https://api.jquery.com/closest/). So in theory, has event delegation in mind while being inclusive of the element that's calling it and working up the tree.